source/kernel/Cargo.toml

```diff
@@ -60,3 +60,9 @@ features = ["derive"]
 [dependencies.postcard]
 version = "1.0.1"
 features = ["experimental-derive"]
+
+[dependencies.embedded-graphics]
+version = "0.7.1"
+
+[dependencies.embedded-graphics-simulator]
+version = "0.3.0"
```

Comment:
Looks like we forgot to remove the actual dependency on `embedded-graphics-simulator` from the kernel?

Reply:
oops! haha missed this one!